### PR TITLE
Update codeFile1.ps1

### PR DESCRIPTION
**Pull request by gnowleumas**

### CODE REVIEW

**Review by sc-samuelwongchangloong — COMMENTED** [review]
Comment 1

**Review by sc-samuelwongchangloong — DISMISSED** [review]
Approve 1

File: src/codeFile1.ps1 [hunk 1]
```diff
@@ -6,4 +6,4 @@ Write-Host "Hello World 5"
 Write-Host "Hello World 6"
 Write-Host "Hello World 7"
 Write-Host "Hello World 8"
-Write-Host "Hello World 11"
```
Comment (sc-samuelwongchangloong) [inline src/codeFile1.ps1:9]: should not do this

File: src/codeFile1.ps1 [hunk 2]
```diff
@@ -6,4 +6,4 @@ Write-Host "Hello World 5"
 Write-Host "Hello World 6"
 Write-Host "Hello World 7"
 Write-Host "Hello World 8"
```
Comment (sc-samuelwongchangloong) [inline src/codeFile1.ps1:8]: what about this?

**Comment by sc-samuelwongchangloong** [comment]
Comment 2

File: src/codeFile1.ps1 (hunk 1 above)
Comment (sc-samuelwongchangloong) [reply src/codeFile1.ps1:9]: ignore this

File: src/codeFile1.ps1 (hunk 2 above)
Comment (gnowleumas) [reply src/codeFile1.ps1:8]: good spot

File: src/codeFile1.ps1 [hunk 3]
```diff
@@ -6,4 +6,4 @@ Write-Host "Hello World 5"
 Write-Host "Hello World 6"
 Write-Host "Hello World 7"
```
Comment (sc-samuelwongchangloong) [inline src/codeFile1.ps1:7]: another review

File: src/codeFile1.ps1 (hunk 2 above)
Comment (gnowleumas) [reply src/codeFile1.ps1:8]: what about now?

**Review by sc-samuelwongchangloong — APPROVED** [review]

**Pull request merged**